native/hdfs/Cargo.toml

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This is an optional HDFS crate
+# To build it from root is required to provide a valid JAVA_HOME
+# and enable `hdfs` feature
+# Example: JAVA_HOME="/opt/homebrew/opt/openjdk@11" cargo build --features=hdfs
+
+[package]
+name = "datafusion-objectstore-hdfs"
+description = "Comet HDFS integration. Initiated by Yanghong Zhong <nju_yaho@apache.org> (https://github.com/datafusion-contrib/datafusion-objectstore-hdfs)"
```

Comment:
I think that it would be better to give credit in https://github.com/apache/datafusion-comet/blob/main/NOTICE.txt

Reply:
+1